### PR TITLE
docs: clarify plugins don't update devtools timeline

### LINKED ISSUE #996: Using `$patch` within a Pinia plugin does not show event in devtools timeline
### Reproduction

https://github.com/Matt-Deacalion/pinia-devtools-bug

### Steps to reproduce the behavior

1. `git clone git@github.com:Matt-Deacalion/pinia-devtools-bug.git`
2. `cd pinia-devtools-bug`
3. `npm install && npm run dev`
4. go to: http://localhost:3000/
5. open vue devtools, see pinia timeline

### Expected behavior

Using `$patch` within a Pinia plugin would show up in Vue devtools within the Pinia timeline.

### Actual behavior

No `$patch` event is shown.

### Additional information

Vue devtools: **6.0.0-beta.21** 
Firefox: **96**
Chromium: **97.0.4692.71**

### PR DESCRIPTION
**Pull request by Matt-Deacalion**
Related to issue: #996

### CODE REVIEW

**Comment by Matt-Deacalion** [comment]
Apologies for the force pushes; no more.

I had issues getting Vitepress to work - I changed [this line](https://github.com/vuejs/pinia/blob/v2/packages/docs/vite.config.ts#L37) to "_../pinia/dist/pinia.esm-**browser**.js_" and it worked fine.

**Comment by Matt-Deacalion** [comment]
Ugh, one more force push. One minute.

File: packages/docs/core-concepts/plugins.md [hunk 1]
```diff
@@ -151,6 +152,8 @@ pinia.use(({ store }) => {
 })
 ```
 
+:::info
+State changes that occur within a plugin are not recorded in the devtools timeline.
```
Comment (posva) [inline packages/docs/core-concepts/plugins.md]: let's move it above the warning and remove the :::info.

**Review by posva — CHANGES_REQUESTED** [review]

File: packages/docs/core-concepts/plugins.md [hunk 2]
```diff
@@ -131,6 +131,8 @@ pinia.use(({ store }) => {
 })
 ```
 
+State changes that occur within a plugin are not recorded in the devtools timeline.
```
Comment (posva) [inline packages/docs/core-concepts/plugins.md]:
```suggestion
Note that state changes or additions that occur within a plugin (that includes calling `store.$patch()`) happen before the store is active and therefore **do not trigger any subscriptions**.
```

**Pull request merged**